app/src/main/kotlin/org/wordpress/aztec/demo/MainActivity.kt

```diff
@@ -170,7 +170,7 @@ open class MainActivity : AppCompatActivity(),
 
         private val isRunningTest: Boolean by lazy {
             try {
-                Class.forName("android.support.test.espresso.Espresso")
+                Class.forName("androidx.test.espresso.Espresso")
```

Comment:
Lost some time to figure out that this was the problem why tests are failing.
Probably we can find some other approach to check if we are testing or not so that other colleagues don't lose too much time on this.
@loremattei @jtreanor @hypest